Source/Lib/Codec/EbPictureAnalysisProcess.c

```diff
@@ -4334,7 +4334,7 @@ void* PictureAnalysisKernel(void *inputPtr)
 		outputResultsPtr->pictureControlSetWrapperPtr = inputResultsPtr->pictureControlSetWrapperPtr;
 
 #if DEADLOCK_DEBUG
-        SVT_LOG("POC %lld PA OUT \n", pictureControlSetPtr->pictureNumber);
+        SVT_LOG("POC %lu PA OUT \n", pictureControlSetPtr->pictureNumber);
 #endif
```

Comment:
should this line be after EbPostFullObject too?

Reply:
Thanks! I thought I've changed it...